framework/helpers/Url.php

```diff
@@ -15,4 +15,31 @@
  */
 class Url extends BaseUrl
 {
+    /**
+     * @param        $params  Array of values to convert into http query
+     * @param string $keyfather Inheritance of keys
+     * @param string $response
+     * @param bool   $ltrim
+     *
+     * @return string
+     */
+    public static function custom_http_build_query($params, $keyfather = '', $response = '', $ltrim = true) {
+        foreach($params as $key => $value) {
+            if(!empty($keyfather)) {
+                if(is_numeric($key)) {
+                    $key = '';
+                }
+                $key = "{$keyfather}[{$key}]";
+            }
+            if(is_array($value)) {
+                $response = Url::custom_http_build_query($value, $key, $response, false);
+            } else {
+                if(!is_null($value)) {
+                    $response .= "&".urlencode($key)."=".urlencode($value);
```

Comment:
Wouldn't it be better to build the `$response` as an array and implode it in the return statement?